.github/workflows/main.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        choiceL: [--disable-silent-rules]
+        choiceL: [--disable-silent-rules, --enable-silent-rules]
```

Comment:
I am curious about the benefit of building assets with and without silent rules? Maybe we can keep only the one you think most appropriate?

Reply:
This is mostly a copy-n-paste from libspiro.
The original intent was to do something like --enable-verbose, but fourier.c didn't access a *.h file
The default is for gcc to run silent, which helps to emphasize errors and warnings, which makes it easy to see, but occasionally you may want to see everything. Some distros, like Debian, generally run showing everything when you look at their reports.
Some people like simple and clean, and for a project of this size, this can come all out, including the workflow if you want to keep it simple. Sometimes, some people want to add extra, but don't know how to go about adding those extras - this can be one of those, I see, I can do something like this :-)
Makefile.am 'make distcheck' probably works okay without the extra options.
If you want a patch to remove it, I'll add one here.